x-pack/test/functional/apps/endpoint/policy_list.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export default function({ getPageObjects, getService }: FtrProviderContext) {
+  const pageObjects = getPageObjects(['common']);
+  const testSubjects = getService('testSubjects');
+
+  describe('Endpoint Policy List', () => {
```

Comment:
I think this is good for an initial set.  We will want to leverage the actual page object file and its capabilities in a future PR.  After my PR is merged it will be easier to refactor.

Reply:
👍 
I might still another test to validate the expected table columns headers are present and display the correct label. Will wait to see again how to use that Table service 😃